smarts/core/local_traffic_provider.py

```diff
@@ -1238,13 +1270,15 @@ def predict_crash_in(self, veh_id: str, bumper: int) -> float:
                 prev_range = rvi.dist
                 prev_bearing = rvi.bearing
                 prev_heading = rvi.heading
-            range_del /= len(window) - 1
-            bearing_del /= len(window) - 1
+            range_del /= len(window) - 1 or 1
+            bearing_del /= len(window) - 1 or 1
```

Comment:
Just checking that the desired outcome when `len(window)=1`, is indeed
```python
range_del = range_del / ( len(window) -1 or 1)
          = range_del                
```

Reply:
Thanks for bringing this up, there is no defined behaviour previously except for a crash. Perhaps it is better to handle the problem explicitly as you mention.

Reply:
Looking at it again I should remove this condition. L1257 covers this case, I believe I should revert this condition.